js/data.js

```diff
@@ -2,7 +2,7 @@
 /* exported updateStoredData */
 
 var data = {
-  view: 'entry-form',
+  view: 'entries',
```

Comment:
You should leave this value as `entry-form`, so users see the form to save entries first, if this is their first visit to the app.